[RFC] vim-patch:{a0f849e, d7464be, b4ff518, e392eb4, d042dc8, 2c5e8e8, 256972a, cc7ff3f}

KillTheMule opened the pull request:
See individual comments, nothing spectacular. Last one is empty (only spell file changes), just there for the scripts to pick up.

Most notably, no change to `runtime/syntax/vim.vim` was applied, it diverged to much from vim.

KillTheMule committed:
vim-patch:a0f849e

Update runtime files.

https://github.com/vim/vim/commit/a0f849ee40cbea3c889345256786b640b0becca2

Missing files runtime/doc/tags and runtime/doc/todo.txt. Excluded
runtime/syntax/vim.vim, since we diverged quite a bit from vim in this file.

KillTheMule committed:
vim-patch:d7464be

Updated runtime files.

https://github.com/vim/vim/commit/d7464be9747fcaa8e6210e1f00a3882932df76e2

Applied cleanly except for runtime/docs/todo.txt and runtime/docs/tags.

KillTheMule committed:
vim-patch:b4ff518

Updated runtime files.

https://github.com/vim/vim/commit/b4ff518d95aa57c2f8c0568c915035bef849581b

Missing files: runtime/doc/tags, runtime/doc/todo.txt. Changes to
runtime/doc/if_pyth.txt, runtime/doc/options.txt and runtime/doc/quickref.txt
did not aply. Excluded runtime/syntax/vim.vim.

KillTheMule committed:
vim-patch:e392eb4

Update runtime files.

https://github.com/vim/vim/commit/e392eb41f8dfc01bd13634e534ac6b4d505326f4

Files runtime/doc/tags and runtime/doc/todo.txt did not exist. Ignored
runtime/syntax/vim.vim. One change in runtime/doc/windows.txt had already been
applied.

KillTheMule committed:
vim-patch:d042dc8

Update runtime files.

https://github.com/vim/vim/commit/d042dc825c9b97dacd84d4728f88300da4d5b6b9

Missing in runtime/doc: hangulin.txt, tags, todo.txt. The changes to options.txt
do not apply for nvim. man.vim is very different in nvim, some changes applied
manually, others discarded.

KillTheMule committed:
vim-patch:2c5e8e8

Updated runtime files.

https://github.com/vim/vim/commit/2c5e8e80eacf491d4f266983f534a77776c7ae83

Missing files in runtime/doc: if_ruby.txt, tags, todo.txt. Ignored changes to
runtime/syntax/vim.vim.

KillTheMule committed:
vim-patch:256972a

Updated runtime files.

https://github.com/vim/vim/commit/256972a9849b5d575b62a6a71be5b6934b5b0e8b

Missing files in runtime/doc: todo.txt, tags. Patch to runtime/doc/syntax.txt
was applied manually in part, for no discernible reason.

KillTheMule committed:
vim-patch:cc7ff3f

Update English spell files.

https://github.com/vim/vim/commit/cc7ff3fcd8c8fd7da6faac98a138b830ec5c00d8

NA for neovim, included so the scripts can pick that up.

justinmk commented:
Thanks, LGTM. There's a small error in the help tags, which fails the builds:

    E154: Duplicate tag ":si" in file ./gui_w32.txt

Just need to delete one of the `:si` tags in there.

KillTheMule committed:
Fix typo that seems to have been around forever

Could not find when it was fixed in the vim source, it originates in 7.001, and
is fixed as of today, but my git-fu did not discern a relevant patch. I don't
think it matters much.

KillTheMule commented:
Fixed.

KillTheMule commented:
Travis build mostly ok, just gcc on osx hangs at input_spec.lua.
